draft-ietf-httpbis-digest-headers.md

```diff
@@ -1059,7 +1061,16 @@ in conjuction with the encrypted content-coding {{?RFC8188}}.
 
 ## Algorithm Agility
 
-...
+Algorithm agility is achieved establishing the IANA Digest Algorithm Values registry,
+and adding it the "Status" field making it possible to deprecate and obsolete insecure algorithms.
+
+Transition from weak algorithms can be achieved
+negotiating the most suitable digest-algorithm using `Want-Digest` (see {{want-digest}})
+or sending multiple representation-data-digest, thus allowing the receiver to select the best
+possible digest-algorithm.
+Even in this case, it is good practice to avoid sending non-necessary
+representation-data-digest values because the receiver could either ignore them (see {{digest}})
+or uselessly consume resources to evaluate all of them.
```

Comment:
Now that there is text, I can see the bones of what you're trying to say. But I think we can rewrite this to be a bit clearer. Here's strawperson text:

"The security properties of digest algorithms are not fixed. It is in an endpoint's best interest to support and use the most secure algorithms that are at its disposal. However, there is little that this specification can do to mandate behaviour. To help endpoints make better algorithm choices a new "Status" field has been added to the IANA Digest Algorithm Values registry (see Section xyz), which contains the most-recent appraisal of the algorithm. Algorithms that have been found insecure or otherwise undesirable can be marked as "deprecated" or "obsolete". Algorithms with no known problems can be marked as "Standard", while new algorithms can be marked "Experimental".

An endpoint might have a preference for algorithms, such as preferring "standard" algorithms over "deprecated" ones. Algorithm agility is supported by negotiation of digest-algorithm using ``Want-Digest` (see {{want-digest}})` or by sending multiple representation-data-digest values for the receiver to choose. Endpoints are advised that sending multiple values consumes resources, which may be wasted if the receiver ignores them (see {{digest}})."